### PR TITLE
Pybnn reqs

### PR DESCRIPTION
**Pull request by marpulli**
*Issue #, if available:* #164 

*Description of changes:* Add pip version of pybnn to reqs and remove skipping of tests.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: requirements/integration_test_requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # For random forest model
 scikit-learn
 # For BNN model
--e git+https://github.com/automl/pybnn.git#egg=pybnn
-# For Lating design
+pybnn==0.0.3
```
Comment (aaronkl) [inline requirements/integration_test_requirements.txt]: could you update that to pybnn==0.0.4? That should fix the unittests

Comment (marpulli) [reply requirements/integration_test_requirements.txt]: Done

Comment (aaronkl) [reply requirements/integration_test_requirements.txt]: I am sorry Mark, I've just found another small bug. Could you set it to pybnn==0.0.5? That fixed at least my local integration tests

Comment (marpulli) [reply requirements/integration_test_requirements.txt]: Will do!

**Pull request merged**